coalib/bears/PackageRequirement.py

```diff
@@ -0,0 +1,22 @@
+from coala_decorators.decorators import generate_repr
+
+
+@generate_repr()
+class PackageRequirement:
```

Comment:
maybe make an own directory, coalib/bears/requirements? This'll grow over time, right? This is the small start of something that may grow really huge over the time